### PR TITLE
Automatically change country via dial code

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
Issue: https://github.com/blackbaud/skyux-phone-field/issues/12

### CODE REVIEW

File: src/app/public/modules/phone-field/phone-field-input.directive.ts [hunk 1]
```diff
@@ -87,8 +87,8 @@ export class SkyPhoneFieldInputDirective implements OnInit, OnDestroy, AfterView
     if (value) {
       let formattedValue = this.formatNumber(value.toString());
 
-      if (this.phoneFieldComponent.selectedCountry.iso2 !== this.phoneFieldComponent.defaultCountry) {
-        formattedValue = '+' + this.phoneFieldComponent.selectedCountry.dialCode +
+      if (this.phoneFieldComponent.selectedCountry.iso2 !== this.phoneFieldComponent.defaultCountry && !value.startsWith('+')) {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/phone-field/phone-field-input.directive.ts]: This condition's readability could be improved by creating a few boolean variables that would help a future contributor parse out what's going on.

Maybe something like:
```
const hasDialCode = (value.startsWith('+'));
const isDefaultCountrySelected = (
  this.phoneFieldComponent.selectedCountry.iso2 ===
  this.phoneFieldComponent.defaultCountry
);

if (!isDefaultCountrySelected && !hasDialCode) {
  ...
}
```

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Feature works great! Just a minor comment regarding code readability.

**Review by Blackbaud-ToddRoberts — APPROVED** [review]
this seems to resolve the corner case bugs i'd seen before

**Review by Blackbaud-AlexKingman — APPROVED** [review]
Works great for me. Ship it.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**